### PR TITLE
chore: update links to Azure IoT Help and Msft Q&A

### PR DESCRIPTION
**Pull request by elhorton**
# Checklist
- [ ] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-java/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.
- I submitted this PR against the correct branch: 
  - [ ] This pull-request is submitted against the `master` branch. 

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

# Description of the solution

### CODE REVIEW

**Comment by elhorton** [comment]
Hi all, these are a few simple changes to our github support workflow for a couple of reasons: 

1) adding reference to Microsoft Q&A, the new, preferred platform for CSS engineers, MVPs, and Microsoft cloud experts to answer questions. We're trying to direct as much traffic on general questions there as possible so they don't show up as Github issues

2) adjusting the link to the official Azure IoT Support page to a vanity link so that we can track how many redirects we have (important for the Azure Customer Experience team to measure success of these workflows). 

Let me know if you have any questions or concerns, thanks!

File: .github/ISSUE_TEMPLATE/technical-question.md [hunk 1]
```diff
@@ -11,12 +11,14 @@ assignees: ''
 
 Thank you for asking a technical question! If your question is "why doesn't my code work?", please see [submit a bug report](https://github.com/Azure/azure-iot-sdk-java/issues/new?assignees=&labels=bug&template=bug-report.md&title=) instead.
 
-We encourage you to submit technical questions related to the SDK on Stack Overflow. We actively monitor all questions tagged with "azure-iot-sdk".
+For quick and reliable answers on your technical product questions from Microsoft Engineers, Azure Most Valuable Professionals (MVPs), or our expert community, engage with us on [Microsoft Q&A](https://aka.ms/azureqa), Azure’s preferred destination for community support. Please tag your question with "azure-iot-sdk".
```
Comment (drwill-ms) [inline .github/ISSUE_TEMPLATE/technical-question.md:14]: I wouldn't capitalize `Engineers`. It isn't a proper noun.

Comment (elhorton) [reply .github/ISSUE_TEMPLATE/technical-question.md]: yes good call, copied and pasted from [this page](https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-support-help) so should get a quick fix there too

**Comment by elhorton** [comment]
hi @timtay-microsoft , could you please help merge this in? seems like the gates are blocked. thank you!

**Comment by timtay-microsoft** [comment]
> hi @timtay-microsoft , could you please help merge this in? seems like the gates are blocked. thank you!

Sure thing! Thanks for the contribution

**Pull request merged**